src/backend/api/Fusion.Resources.Api/appsettings.json

```diff
@@ -22,8 +22,9 @@
   },
   "Fusion": {
     "ServiceDiscovery": {
-      //"ProOrganisation": "https://localhost:44311"
-      //"people": "https://localhost:44332"
+      "ProOrganisation": "https://pro-s-org-fqa.azurewebsites.net",
```

Comment:
Any specific reason for using QA (or maybe CI) instead of local environment settings ?

Reply:
Just for local testing. Commented these out, using default settings.